.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "java.configuration.updateBuildConfiguration": "automatic"
+}
```

Comment:
[MINOR] .vscode folder should be added to gitignore

Ref: https://github.com/github/gitignore , good resource for starter gitignore files.